tinylog-impl/src/main/java/org/tinylog/pattern/TimestampToken.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 Martin Winandy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package org.tinylog.pattern;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.Collections;
+
+import org.tinylog.core.LogEntry;
+import org.tinylog.core.LogEntryValue;
+
+/**
+ * Token for outputting the time of issue of a log entry as a Unix timestamp.
+ */
+final class TimestampToken implements Token {
+
+	private static final long SECONDS_DIVISOR = 1000;
+
+	private final boolean useSeconds;
+
+	/**	*/
+	TimestampToken() {
+		this.useSeconds = false;
+	}
+
+	/**
+	 * @param unit
+	 *            Unit of timestamp (e.g. seconds)
+	 *            Invalid value defaults to milliseconds.
+	 */
+	TimestampToken(final String unit) {
+		this.useSeconds = "seconds".equals(unit);
```

Comment:
Could you test for "milliseconds" to ensure that seconds are default fallback unit?